pkg/controller/atlasproject/atlasproject_controller.go

```diff
@@ -133,6 +133,11 @@ func (r *AtlasProjectReconciler) Delete(e event.DeleteEvent) error {
 	log := r.Log.With("atlasproject", kube.ObjectKeyFromObject(project))
 	log.Infow("-> Starting AtlasProject deletion", "spec", project.Spec)
 
+	if project.Status.ID == "" {
+		log.Infof("Project does not exist, nothing to remove")
```

Comment:
[nit] maybe we could add "does not exist in Atlas"?

Reply:
I agree, better to be concrete 👌